src/API/users/checkin.js

```diff
@@ -1,70 +1,73 @@
-const logger = require('../../log/logger');
-const checkin = require('../../data/schema/checkin-schema');
-const users = require('../../data/schema/user-schema');
-const moment = require('moment');
+const logger = require("../../log/logger");
+const checkin = require("../../data/schema/checkin-schema");
+const users = require("../../data/schema/user-schema");
+const moment = require("moment");
+const { validationResult } = require("express-validator");
 
 module.exports = function(req, res) {
+  const errores = validationResult(req);
+  if (!errores.isEmpty())
+    return res.status(422).json({ errores: errores.array() });
```

Comment:
We should use only english I will suggest use "error" instead of "errores"

Reply:
done